CMakeLists.txt

```diff
@@ -116,11 +116,18 @@ include(GNUInstallDirs)
 set(concurrencpp_directory "concurrencpp-${PROJECT_VERSION}")
 set(concurrencpp_include_directory "${CMAKE_INSTALL_INCLUDEDIR}/${concurrencpp_directory}")
 
-install(TARGETS concurrencpp
-        EXPORT concurrencppTargets
-        ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT concurrencpp_Development
-        RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}" COMPONENT concurrencpp_Development
-        INCLUDES DESTINATION "${concurrencpp_include_directory}")
+install(
+  TARGETS concurrencpp
+  EXPORT concurrencppTargets
+  ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+          COMPONENT concurrencpp_Development
+  INCLUDES
+  DESTINATION "${concurrencpp_include_directory}"
+  COMPONENT concurrencpp_Development
+  LIBRARY DESTINATION "${CMAKE_INSTALL_BINDIR}"
```

Comment:
[`LIBRARY`](https://cmake.org/cmake/help/latest/command/install.html#:~:text=with%20ENABLE_EXPORTS%20enabled.-,LIBRARY,-Target%20artifacts%20of):

> Target artifacts of this kind include:
> - Shared libraries, except
>   - DLLs (these go to RUNTIME, see below),

Such things go in `CMAKE_INSTALL_LIBDIR` instead of `CMAKE_INSTALL_BINDIR`.

Reply:
Yes, you are right. I just forgot to fix it when copy-pasting :-).